<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMLinux GUI Config Window Thingy by @bs2k#2511</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="QWidget" name="mainTab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Preset</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="opt_PRESET">
           <item>
            <property name="text">
             <string>sm64pc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sm64ex</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>render96ex</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>r96proto</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>cheaterex</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sm64-port</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sm64nx</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sm64ex-coop</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sm64dos</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sm64-portweb</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sm64exweb</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sm64-port-android-base</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>androidex</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_InstallHD">
         <property name="text">
          <string>Install HD</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_InstallR96">
         <property name="text">
          <string>Install Render96</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_FPS60">
         <property name="text">
          <string>60FPS</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Version</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="opt_VERSION">
           <item>
            <property name="text">
             <string>us</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>jp</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>eu</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="do_MAXJOBS">
         <property name="text">
          <string>Limit Job Count</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Max Job Count</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="opt_MAXJOBS">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sm64exTab">
      <attribute name="title">
       <string>sm64ex &amp;&amp; forks</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="opt_CHEATER">
         <property name="text">
          <string>CHEATER</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_BETTERCAMERA">
         <property name="text">
          <string>Puppycam</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_NODRAWINGDISTANCE">
         <property name="text">
          <string>No Draw Distance</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_TEXTURE_FIX">
         <property name="text">
          <string>Texture Fix</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_EXTERNAL_DATA">
         <property name="text">
          <string>External Data</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_DISCORDRPC">
         <property name="text">
          <string>Discord Rich Presence</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Render API</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="opt_RENDER_API">
           <item>
            <property name="text">
             <string>GL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GL_LEGACY</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_TEXTSAVES">
         <property name="text">
          <string>Text Saves</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sm64excoopTab">
      <attribute name="title">
       <string>sm64ex-coop</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="opt_DISCORD_SDK">
         <property name="text">
          <string>Discord SDK</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_IMMEDIATELOAD">
         <property name="text">
          <string>Immediate Load</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>397</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="render96exTab">
      <attribute name="title">
       <string>render96ex</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="opt_LEGACY_RES">
         <property name="text">
          <string>Legacy Resources</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>436</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="androidTab">
      <attribute name="title">
       <string>Android</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QCheckBox" name="opt_TOUCH_CONTROLS">
         <property name="text">
          <string>Touch Controls</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_ARMONLY">
         <property name="text">
          <string>ARM Only</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>397</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="DOSTab">
      <attribute name="title">
       <string>DOS</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QCheckBox" name="opt_ENABLE_OPENGL_LEGACY">
         <property name="text">
          <string>OpenGL Legacy</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>GL</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="opt_DOS_GL">
           <item>
            <property name="text">
             <string>dmesa</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>osmesa</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>400</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QCheckBox" name="opt_CONFIG">
         <property name="text">
          <string>Config</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Base Path</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="opt_BASEPATH">
           <property name="text">
            <string>$HOME</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_UpdateHD">
         <property name="text">
          <string>Update Addons</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_DEBUG">
         <property name="text">
          <string>Debug</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_TARGET_WEB">
         <property name="text">
          <string>Target Web</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_TARGET_RPI">
         <property name="text">
          <string>Target Raspberry Pi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_OSX_BUILD">
         <property name="text">
          <string>macOS Build</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="do_GIT">
         <property name="text">
          <string>Custom Repo</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Git Repo</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="opt_GIT">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>sm64pc/sm64ex</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Branch</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="opt_BRANCH">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>nightly</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_9">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Linux</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="opt_Linux">
           <property name="text">
            <string>sudo apt install -y build-essential bsdmainutils wget git python3 libaudiofile-dev libglew-dev libsdl2-dev binutils libusb-1.0-0-dev libzstd-dev python3-pip mplayer zenity</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="opt_AutoUpdate">
         <property name="text">
          <string>Auto Update</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>19</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
